api_tests/lib/ledger_runner.ts

```diff
@@ -171,15 +171,8 @@ async function startBitcoinContainer(): Promise<StartedTestContainer> {
 }
 
 async function startEthereumContainer(): Promise<StartedTestContainer> {
-    return new GenericContainer("parity/parity", "v2.5.0")
-        .withCmd([
-            "--config=dev",
-            "--jsonrpc-apis=all",
```

Comment:
Reposting my comment here since it got ignored in the other thread:

We do we no longer pass all the other configuration options?

Reply:
cc @bonomat

Reply:
Tried to answer this in the other thread :) : 
All the configs are set in the config file on default. So there is no need to pass in them here again.

Reply:
Alright, I misunderstood, thanks!
I am not 100% if that is good but since any PoA network is probably used for testing, it should be fine.

Reply:
It might be a bit surprising to users so we should probably mention it in the README of the Docker container. At the moment, it only says that it is standard parity + PoA but actually it is standard parity + PoA + certain logging + exposed APIs.

Reply:
Fair, updated the docker readme.